### ISSUE TITLE
Error: no accessibility issues found

### ISSUE THREAD

**Issue by dippedrusk** [netlify-labs/netlify-plugin-a11y#68]
**Describe the bug**
No accessibility issues were found but the plugin still errors out and fails my build haha

**To Reproduce**
1. Write code with no accessibility issues that trigger this plugin
2. Run the plugin with the following netlify.toml config
```
[[plugins]]
  package = "@netlify/plugin-a11y"
  [plugins.inputs]
    wcagLevel = 'WCAG2AAA'
```

**Expected behavior**
I expect it to pass!

**Screenshots**
![Screen Shot 2021-11-24 at 12 54 33 PM](https://user-images.githubusercontent.com/15020857/143234060-3f454748-f9e7-4278-a1a4-259f25d1890e.png)

**Desktop (please complete the following information):**
Not applicable

**Smartphone (please complete the following information):**
Not applicable

**Additional context**
Here's the public deploy log: https://app.netlify.com/sites/naughty-wilson-684740/deploys/619e2798f3af5100098524f8
